stage1/init/common/pod.go

```diff
@@ -330,12 +470,12 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		opts = append(opts, unit.NewUnitOption("Service", "SyslogIdentifier", filepath.Base(app.Exec[0])))
 	}
 
-	if flavor == "kvm" {
-		capabilities := append(app.Isolators, nspawnCapabilities.AsIsolator())
-		capabilitiesStr := GetAppCapabilities(capabilities)
+	// TODO: read the RemoveSet as well. See
```

Comment:
Since this is no longer hidden in a conditional, you can set it up before initalising opts on line 313 (449) as you do with everything else